docs/zh/react/reactlynx-testing-library/index.mdx

```diff
@@ -420,7 +345,11 @@ test('it waits for the data to be loaded', async () => {
 });
 ```
 
-### Rerender
+在这个例子中，我们使用 `waitForElementToBeRemoved` 方法来等待 _Loading..._ 元素被移除，此时页面中会渲染出 _Loaded this message: Hello World!_ 元素。这时我们可以用 `screen.getByTestId` 方法来获取页面中的元素，并断言其文本内容是否正确。
+
+### rerender
+
+`render` 方法返回的对象中包含 `rerender` 方法，可以用于重新渲染页面。`rerender` 方法会将新的组件渲染到页面中，并返回一个新的对象。可以使用 `rerender` 方法来测试组件的不同状态。
 
 ```jsx
```

Comment:
You should mention "   // 不同于 React Testing Library，container 需要在 rerender 之后重新获取
    // 因为 ReactLynx 每次加载都会创建一个新的 page 元素"  in the main paragraph as a warning (highlight block) rather than in comment as this is important